packages/design-system-docs/src/pages/components/FilterChip/FilterChip.example.jsx

```diff
@@ -35,17 +35,13 @@ ControlledFilterChip.propTypes = {
 };
 
 ReactDOM.render(
-  <>
+  <div aria-live="polite" aria-relevant="removals">
     <ControlledFilterChip label="Example Filter Chip" ariaClearLabel="Remove Filter Chip" />
     <ControlledFilterChip
       label="Example with alternate icon"
       useAlternateIcon
       ariaClearLabel="Remove Alternate"
     />
-    <ControlledFilterChip
-      label="Example filter chip with long text that will wrap Example filter chip with long text that will wrap Example filter chip with long text that will wrap Example filter chip with long text that will wrap Example filter chip with long text that will wrap"
-      ariaClearLabel="Remove Long"
-    />
     <ControlledFilterChip
       label="Example big filter chip"
       ariaClearLabel="Remove Small"
```

Comment:
This clear label value doesn't really make sense either

Reply:
In general these examples don't really match how they'd be used in the real world. I think usually the contents would be much more succinct when used as filters.

Reply:
Like it's not super clear how the `ariaClearLabel` should relate to the content of the filter

Reply:
Oh yeah, I don't think I reviewed the language in our current examples. I think I need direction on what the expected readout for this label is before I can address.